.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ orbs:
 
 aliases:
   # Docker image tags can be found here: https://circleci.com/developer/images/image/cimg/android
-  - &cimg cimg/android:2024.04.1-node
+  - &cimg cimg/android:2025.01.1-node
```

Comment:
Doing the switch got us over the [setup environment error](https://app.circleci.com/pipelines/github/forcedotcom/SalesforceMobileSDK-Android/3220/workflows/fcdd3f55-b9b2-41a1-a402-8088ba7d3a89/jobs/28398/parallel-runs/0/steps/0-106)

Reply:
Also latest image has android-35 (but drops android-28).